conf.pri.mingw

```diff
@@ -0,0 +1,43 @@
+# Adapt these paths
+# Point these to the include folders
+INCLUDEPATH += $$quote(C:/qBittorrent/boost_1_51_0)
+INCLUDEPATH += $$quote(C:/qBittorrent/RC_0_16/include)
+INCLUDEPATH += $$quote(C:/qBittorrent/Zlib/include)
+INCLUDEPATH += $$quote(C:/qBittorrent/openssl/include)
+# Point these to the lib folders
+LIBS += $$quote(-LC:/qBittorrent/boost_1_51_0/stage/lib)
+LIBS += $$quote(-LC:/qBittorrent/RC_0_16/bin/path-according-to-the-build-options-chosen)
+LIBS += $$quote(-LC:/qBittorrent/Zlib/lib)
+LIBS += $$quote(-LC:/qBittorrent/openssl/lib)
+
+# Adapt the lib names/versions accordingly
+CONFIG(debug, debug|release) {
+    LIBS += libtorrent-rasterbar \
+            libboost_system-mt \
+            libboost_filesystem-mt \
+            libboost_thread_win32-mt
+}
+else {
+    LIBS += libtorrent-rasterbar \
+            libboost_system-mt \
+            libboost_filesystem-mt \
+            libboost_thread_win32-mt
+}
+
+LIBS += libadvapi32 libshell32 libuser32 libole32
+LIBS += libcrypto libssl libwsock32 libws2_32 libz libiconv
+LIBS += libpowrprof
+
+DEFINES += NTDDI_VERSION=0x05010000
+DEFINES += _WIN32_WINNT=0x0501
+DEFINES += _WIN32_IE=0x0501
+
+DEFINES += BOOST_ALL_NO_LIB
+# Boost 1.60+ defaults to Vista+ support. The define below enables XP support again.
+DEFINES += BOOST_USE_WINAPI_VERSION=0x0501
+# Disable this if building against libtorrent 1.1.x (RC_1_1)
+DEFINES += BOOST_ASIO_SEPARATE_COMPILATION
+DEFINES += BOOST_SYSTEM_STATIC_LINK=1
+
+# Enable this when building against libtorrent 1.1.x (RC_1_1)
```

Comment:
1.1 is the default for 3.4. Please change uncomment the next line and change the comment appropriately.

Reply:
OK.